k8s/demo/common/divorce/div-cos.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: divorce
   annotations:
     flux.weave.works/automated: "true"
-    flux.weave.works/tag.java: glob:prod-*
+    flux.weave.works/tag.java: glob:pr-1054-*
```

Comment:
why?

Reply:
To get PR build to demo. Not deployed on AAT yet